packages/docmanager-extension/src/index.ts

```diff
@@ -473,7 +473,7 @@ function addCommands(
       }
 
       return docManager.services.contents.getDownloadUrl(path).then(url => {
-        window.open(url, '_blank');
+        window.open(url, '_blank', 'noopener');
```

Comment:
I think what needs to happen is that the window return by your `window.open(...)` call needs to have its `opener` attribute set to `null`, because I don't think the third argument takes `noopener`.
```typescript
const opened = window.open();
opened.opener = null;
opened.location = url;
```

cf. https://mathiasbynens.github.io/rel-noopener/

Reply:
Yes, this should be a safer way. Another thing: is `docManager.services.contents.getDownloadUrl` can be treated as safe? thus no need to consider the issue 5655?

Reply:
Hi @afshin , for more secure, I've add the lines in your comment, see the new commit.